src/Components/Carousel/Carousel.js

```diff
@@ -61,17 +94,25 @@ const Carousel = () => {
                     <img src={e.hdurl} className="ImageWrapper"/>
                
                 <h3 className="ImageTittle">{e.title}</h3>
-                <p className="ImageText">{e.explanation}</p>
+               <button id="text" onClick={()=> showModal(e)}>TEXT</button>
                
                 </div> 
                 )
         })}
         </div>
-        
             <div className="RightButton">
                 <button id="forward" onClick={goForward}>⇨</button>
             </div>
         </div>
+        <div className="Buttons">
+        {activePage === 0 ? <button className="active"></button> : <button className="btnStyle"></button>}
+        {activePage === 1 ? <button className="active"></button> : <button className="btnStyle"></button>}
+        {activePage === 2 ? <button className="active"></button> : <button className="btnStyle"></button>}
+        </div>
+        <Modal modal={modal}
```

Comment:
Let's write props in new lines and closing tag to be in line with opening tag.
            <Modal
                modal={modal}
                data={text}
                handleClose={closeModal}
            />